app/src/main/res/layout-sw600dp-port/question_player_fragment.xml

```diff
@@ -120,36 +147,6 @@
         tools:layout_constraintGuide_percent="0.5f" />
     </androidx.constraintlayout.widget.ConstraintLayout>
 
-    <FrameLayout
```

Comment:
Add this back.

Reply:
@aishapervin Apologies for the confusion here, I thought it isn't needed here. Thanks, @rt4914

Reply:
changes are done